manifest.json

```
@@ -1354,6 +1354,19 @@
               }
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/opensuse/15.2/helix/amd64",
+              "os": "linux",
+              "osVersion": "leap",
```

```suggestion
              "osVersion": "leap15.2",
```

And then I'll need to make another change to Image Builder to support a display name for this.

---

https://github.com/dotnet/docker-tools/pull/874